config/foreman-proxy-content.migrations/210112194603-enable-pulp3-content-proxy.rb

```diff
@@ -0,0 +1,7 @@
+answers.delete('foreman_proxy::plugin::pulp') if answers.key?('foreman_proxy::plugin::pulp')
+
+if answers['foreman_proxy_content'].is_a?(Hash)
+  answers['foreman_proxy_content']['pulpcore_mirror'] = true
+  answers['foreman_proxy_content'].delete('proxy_pulp_isos_to_pulpcore')
+  answers['foreman_proxy_content'].delete('proxy_pulp_yum_to_pulpcore')
```

Comment:
In https://github.com/theforeman/puppet-foreman_proxy_content/pull/306 you've removed these parameters. Kafo should already do that automatically. There is a migration where I made the same mistake, but I don't think you need code for this.
```suggestion
```